NetflixClone/Classes/Scenes/Home/HomeViewModel.swift

```diff
@@ -0,0 +1,58 @@
+//
+//  HomeViewModel.swift
+//  NetflixClone
+//
+//  Created by Joseph Ching on 2022-10-08.
+//
+
+import Foundation
+import Combine
+
+class HomeViewModel {
+    
+    // MARK: - Properties
+    private let remote = DiscoverMoviesRemote()
+    private var hasMoreMovies = true
+    
+    var cancellables: Set<AnyCancellable> = []
+        
+    @Published var result: NewMovies? = NewMovies(newMovies: [], section: .featured)
```

Comment:
```suggestion
        
    @Published private(set) var result = NewMovies(newMovies: [], section: .featured)
```

Reply:
Will make sure that variables like this will be private(set) from now on 👍.